src/Symfony/Component/Yaml/Inline.php

```diff
@@ -669,6 +669,10 @@ private static function parseTag(string $value, int &$i, int $flags): ?string
         $nextOffset = $i + $tagLength + 1;
         $nextOffset += strspn($value, ' ', $nextOffset);
 
+        if ('' === $tag && (!isset($value[$nextOffset]) || \in_array($value[$nextOffset], [']', '}', ','], true))) {
+            throw new ParseException(sprintf('Using the unquoted scalar value "!" is not supported. You must quote it at line 1 (near "%s")', $value), self::$parsedLineNumber + 1, $value, self::$parsedFilename);
```

Comment:
missing dot at the end of the message

Reply:
Done. I actually don't need to add the `at line 1 (near "%s")` since it's added by the exception.